packages/scene-composer/src/components/RadioButton/radio-button.scss

```diff
@@ -1,13 +1,33 @@
+@mixin radio {
+  appearance: none;
+  background-color: #fff;
+  color: #fff;
+  border: 4px solid #00a1c9;
+  border-radius: 50%;
+  padding: 3px;
+}
+
 input[type="radio"] {
   margin-right: 1rem;
 
   // styled to match Polaris: https://polaris.a2z.com/components/radio-group/
   &:checked {
-    appearance: none;
-    background-color: #fff;
-    color: #fff;
-    border: 4px solid #00a1c9;
-    border-radius: 50%;
-    padding: 3px;
+    @include radio;
+  }
+}
+
+// Grafana override
```

Comment:
I don't think we should mention Grafana in here or in commit/PR message. It is just one of the AppKit client, but this would fix the override by all clients

Reply:
Fixed